gh-126165: Improve docs of function `math.isclose` (#126215)
python/cpython: Doc/library/math.rst

   between *a* and *b*, relative to the larger absolute value of *a* or *b*.
   For example, to set a tolerance of 5%, pass ``rel_tol=0.05``.  The default
   tolerance is ``1e-09``, which assures that the two values are the same

>    within about 9 decimal digits.  *rel_tol* must be a nonnegative value.

Reviewer: No, I don't think that nonnegative value for rel_tol does make sense.  Rather we should adjust implementation.

Author: Sorry, I didn't read your previous comment clearly.

   tolerance is ``1e-09``, which assures that the two values are the same
   within about 9 decimal digits.  *rel_tol* must be greater than zero.

>    *abs_tol* is the absolute tolerance; it must be at least ``0.0``.

Reviewer 2: Ditto

>    *abs_tol* is the minimum absolute tolerance; it must be at least ``0.0``.
>    When comparing to ``0.0``, the default *abs_tol* of ``0.0`` means that only
>    ``0.0`` is close to ``0.0``. Pass a larger tolerance to allow for floating
>    point inaccuracies.

Reviewer: ```suggestion
   *abs_tol* is the minimum absolute tolerance; it must be at least ``0.0``.
   When comparing to ``0.0``, the default tolerance values means that only
   ``0.0`` is close to ``0.0``.  Pass an appropriate absolute tolerance to
   compare with ``0.0``.
```

BTW, should we keep "minimum"?

Author: I don't think it's needed.
